config.ru

```diff
@@ -21,7 +21,9 @@ end
 if Builds::BUILD_CONFIG['teamCityBaseUrl']
   require 'teamcity'
   TeamCity.configure do |config|
-    config.endpoint = Builds::BUILD_CONFIG['teamCityBaseUrl'] + '/app/rest?guest=1'
+    config.endpoint = Builds::BUILD_CONFIG['teamCityBaseUrl'] + '/app/rest'
+    config.http_user = ENV['TC_USER']
+    config.http_password = ENV['TC_PASSWORD']
```

Comment:
Should the two new lines be in an `if` statement? I'm not sure what the TeamCity gem does if you specify a `nil` username or password?

Reply:
Yeah sounds like an improvement. I wonder if we should switch on guest user?

Reply:
I am on holiday but can make a deep dive and change when I am back? Or you are welcome to improve this also :) if you have time

Reply:
fixed. Now both guest and password authentication works (tested).